In [55]:
number = []
total = 0
with open('input.txt') as f:
    for line in f:
        if line != '\n':
            total = int(total) + int(line)
        if line == '\n':
            number.append(total)
            total = 0
            continue
print(number)
        

            

[55224, 45594, 46528, 44989, 34883, 47314, 54811, 39739, 48715, 36498, 59358, 47256, 49490, 32390, 53552, 62791, 51018, 44613, 26977, 55729, 36703, 52724, 49400, 52228, 62021, 58450, 41118, 16255, 17720, 52587, 45494, 53159, 68718, 51258, 50144, 73211, 63076, 28182, 64633, 22395, 42403, 57557, 53211, 41464, 40622, 49758, 22547, 53980, 60472, 44318, 36225, 47783, 60203, 52279, 35853, 38086, 43064, 13898, 16712, 50811, 36448, 52618, 41155, 38938, 71575, 45157, 57251, 26439, 7352, 48262, 49282, 46832, 54097, 26007, 45614, 57201, 37048, 56991, 49680, 58789, 52965, 47277, 42014, 52789, 42406, 60538, 34325, 35682, 48581, 24135, 9083, 56077, 51899, 61092, 57901, 38273, 44476, 56784, 32633, 51247, 63021, 53442, 46865, 42598, 60197, 35817, 57499, 61462, 30461, 57736, 29166, 55676, 32323, 56357, 46350, 59170, 42438, 33188, 47928, 53692, 66910, 12946, 43167, 37717, 32591, 44884, 47317, 52804, 54999, 46832, 49506, 48515, 46546, 43682, 35676, 32859, 37023, 57280, 50777, 57844, 59113, 47347, 43036, 

In [56]:
print(max(number))

73211


73211 is the answer to the first question!
Let's continue with part two

In [60]:
sum(sorted((number), reverse=True)[:3])

213958